### PR TITLE
HDDS-4581. Cleanup usage of volumeArgs in KeyRequests.

### PR DESCRIPTION
**Pull request by bharatviswa504**
## What changes were proposed in this pull request?

Cleanup usage of VolumeArgs in Request/Response classes in Key requests.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-4581

## How was this patch tested?

Existing tests.

### CODE REVIEW

File: hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/file/OMFileCreateRequest.java [hunk 1]
```diff
@@ -279,7 +279,6 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
           .collect(Collectors.toList());
       omKeyInfo.appendNewBlocks(newLocationList, false);
 
-      omVolumeArgs = getVolumeInfo(omMetadataManager, volumeName);
```
Comment (linyiqun) [inline hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/file/OMFileCreateRequest.java:282]: Why we not remove omVolumeArgs variable defined in this method?

File: hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeyCommitRequest.java [hunk 2]
```diff
@@ -193,7 +193,6 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
 
       long scmBlockSize = ozoneManager.getScmBlockSize();
       int factor = omKeyInfo.getFactor().getNumber();
-      omVolumeArgs = getVolumeInfo(omMetadataManager, volumeName);
       omBucketInfo = getBucketInfo(omMetadataManager, volumeName, bucketName);
```
Comment (linyiqun) [inline hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeyCommitRequest.java:196]: Same comment for above.

**Review by linyiqun — COMMENTED** [review]
Few minor comments from me.
In additional, with compared the change in HDD-4308. S3MultipartUploadAbortRequest/TestS3MultipartUploadAbortRequest is missed updated here.

**Comment by bharatviswa504** [comment]
Thank You @linyiqun for the review, addressed review comments in the latest update.

File: hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/file/OMFileCreateRequest.java (hunk 1 above)
Comment (bharatviswa504) [reply hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/file/OMFileCreateRequest.java:282]: I missed it, thanks for catching it.

**Comment by linyiqun** [comment]
@bharatviswa504 , would you mind checking if the failed unit related to this PR? If not, I am +1 for this PR.

File: hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/file/OMFileCreateRequest.java [hunk 3]
```diff
@@ -279,7 +277,6 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
...
```
Comment (captainzmc) [inline hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/file/OMFileCreateRequest.java:282]: All the use of the getVolumeInfo method has been removed, so getVolumeInfo should not be needed, we can [delete it.]( https://github.com/apache/ozone/blob/master/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeyRequest.java#L620)

**Review by captainzmc — COMMENTED** [review]
LGTM over all, just one comment. CI failure may have nothing to do with this PR and we can retrigger it.

File: hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/file/OMFileCreateRequest.java (hunk 3 above)
Comment (bharatviswa504) [reply hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/file/OMFileCreateRequest.java:282]: Done

**Review by captainzmc — APPROVED** [review]
+1 LGTM.

**Review by linyiqun — APPROVED** [review]
+1.

**Comment by captainzmc** [comment]
Thanks for @bharatviswa504’s patch. Also thanks for the review of @linyiqun. Merged this.

**Pull request merged**